fixtures/istanbul-should-cover.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
```

Comment:
Let's move the fixture into the test file (and narrow as much of the contents as possible to still reproduce the errors)